os/tools/mksmartfsimg.sh

```diff
@@ -21,12 +21,18 @@
 
 source .config
 
-BOARDNAME=$CONFIG_ARCH_BOARD
+BOARDNAME=artik05x
 POSTFIX=_smartfs.bin
+BINDIR=../build/output/bin
 BINNAME=$BOARDNAME$POSTFIX
 CONTENTSDIR=../tools/fs/contents
+NXFUSEDIR=../tools/nxfuse
```

Comment:
1. We don't have the nxfuse in tools folder.
2. I already told you that we should rename it from nx.